### PR TITLE
Change files to be inline with coding conventions and other small changes

### PR DESCRIPTION
**Pull request by clemwj**
Leave out export and system for now

### CODE REVIEW

**Comment by MaxLisaius** [comment]
In the coding conventions, "CamelCase" is used, however as I have seen it camel case is used as "camelCase". We may want to choose a different term to describe the naming of the "FirstSecond" style in namespaces and classes.

**Comment by clemwj** [comment]
> In the coding conventions, "CamelCase" is used, however as I have seen it camel case is used as "camelCase". We may want to choose a different term to describe the naming of the "FirstSecond" style in namespaces and classes.

Good idea. Looking at camelCase vs CamelCase, they can both use the same name. I'll change it to Pascal Case (another name for FirstSecond version) to avoid confusion.

File: program/src/main/Question.vala [hunk 1]
```diff
@@ -2,50 +2,49 @@ using Xml;
...
         }
-        
-
-        Save.ret_to_ex (saver.end_element());
-        
-    }
-    
-    public Gee.ArrayList<int> getMarks(){
-        return this.marks;
+        Save.ret_to_ex (saver.end_element());
```
Comment (MaxLisaius) [inline program/src/main/Question.vala:48]: I am confused by the area removed versus the added code here, could you clarify on the change here?

**Comment by MaxLisaius** [comment]
We may want to simplify our syntax for getters and setters, I think that "public string automatic { get; set; }" should work for a lot of out get and set uses. https://wiki.gnome.org/Projects/Vala/PropertiesSample
This could improve readability in files like Mark.vala

File: program/src/main/Question.vala (hunk 1 above)
Comment (clemwj) [reply program/src/main/Question.vala:48]: Not sure why it says the Save is new.... but I moved the getter next to the other getters in the file (line 22)

**Comment by clemwj** [comment]
> We may want to simplify our syntax for getters and setters, I think that "public string automatic { get; set; }" should work for a lot of out get and set uses. https://wiki.gnome.org/Projects/Vala/PropertiesSample
> This could improve readability in files like Mark.vala

I like this idea. Are you thinking that it should be a part of this PR?

**Comment by MaxLisaius** [comment]
> > We may want to simplify our syntax for getters and setters, I think that "public string automatic { get; set; }" should work for a lot of out get and set uses. https://wiki.gnome.org/Projects/Vala/PropertiesSample
> > This could improve readability in files like Mark.vala
> 
> I like this idea. Are you thinking that it should be a part of this PR?

I think we can move this to another PR if we want, we don't have to do this now. Whatever is the most clean workflow.

**Comment by MaxLisaius** [comment]
Looks Good

**Pull request merged**